ravindu14/SliitSpmProject
# The registration form details stored to the database

**How to test**

1. Checkout to the branch feature/task-1024-store-the-registration-details
2. Fill some fileds and keep some empty.
3. Then error messages should be poped up.
4. Then fill all the fields and click on register button. Then values should be go to the database.
5. When cancel button or registration button is clicked the form should be initialized.
